library/src/test/java/com/datatorrent/lib/io/jms/SQSStringInputOperatorTest.java

```diff
@@ -71,47 +77,85 @@
     JMSStringInputOperator operator;
     CollectorTestSink<Object> sink;
     Context.OperatorContext context;
+    String accessKeyId = "";
+    String secretKey = "";
     SQSTestBase testBase;
 
+    private void createElasticMQMockServer()
+    {
+      // just use x and y as access and secret keys respectively: not checked by the mock server
```

Comment:
Suggest a similar explanatory comment on accessKeyId and secretKey which are initialized to empty strings above.